demos/serverless/README.md

```diff
@@ -29,3 +29,8 @@ node ./deploy.js -r us-east-1 -b <my-bucket> -s <my-stack-name> -a meeting
 The script will create an S3 bucket and CloudFormation stack
 with Lambda and API Gateway resources required to run the demo. After the script
 finishes, it will output a URL that can be opened in a browser.
+
+
+### Cleaning up
+To avoid incurring future charges, please delete any resources in your account that you are not using. 
+You can delete the CloudFormation stack, which will clean up all associated resources
```

Comment:
- nit: missing `.` at the end of a sentence
- How about linking CF documentations? e.g.

  You can delete the provisioned CloudFormation stack using the [AWS CloudFormation console](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-delete-stack.html) or the [AWS CLI](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-cli-deleting-stack.html).

Reply:
Updated.